### PR TITLE
Lab11

### PR DESCRIPTION
**Pull request by Andrew174194**
Pull request for lab 11
by Andrey Khoroshavin (kill team, CS-01 academic group)

### CODE REVIEW

**Comment by Andrew174194** [comment]
Thanks for LAB10.md in 11th lab :)

**Review by DOCtorActoAntohich — APPROVED** [review]
gud job

File: k8s/11.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Secret with Helm's `secrets.yaml`
+
+Let me create the environment variable for particular Helm cluster. For this puropses I define encoded to `base64` secret token inside the [secrets.yaml](./chart-msc-time/templates/secrets.yaml) and include it into a cluster with `spec.template.spec.containers.env` property in [deployment.yaml](./chart-msc-time/templates/deployment.yaml).
+
+As a result, the presence of variable inside the Helm cluster containers can be checked by the part of output of the following command:
+```sh
+$ kubectl describe pod
+Name:             msc-time-py-554db79d97-rdqjz
+...
+    Limits:
```
Comment (ntdesmond) [inline k8s/11.md:50]: ????
where env variable

File: k8s/chart-msc-time/templates/secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: top-token
+  namespace: default
+data:
+  token: QW4wdGhFUl9UMGtlTg==
```
Comment (ntdesmond) [inline k8s/chart-msc-time/templates/secrets.yaml:7]: ah so its no plugin
cool, still

**Review by ntdesmond — APPROVED** [review]

**Pull request closed without merging**